book/myst.yml

```diff
@@ -25,6 +25,7 @@ project:
         - file: tutorials/intro.md
         - file: tutorials/parallel-pipeline.md
         - file: tutorials/use-the-artifact-cache.md
+        - file: tutorials/self-signing-results.md
```

Comment:
This kinda feels like a How-to to me, especially since we don't have any tutorial data to mess with.